Makefile.js

```diff
@@ -3,8 +3,8 @@
  * @author nzakas
  */
 
-/* global target */
-/* eslint no-use-before-define: "off", no-console: "off" */
+/* global target -- How is this being set? */
```

Comment:
Could possibly use a `TODO` prefix comment to indicate this needs to be updated.

```suggestion
/* global target -- TODO: figure out where this is coming from and update this description */
```